pinot-integration-tests/src/test/java/org/apache/pinot/integration/tests/OfflineClusterIntegrationTest.java

```diff
@@ -355,9 +354,10 @@ public void testInvertedIndexTriggering()
       }
     }, 600_000L, "Failed to cleanup obsolete index");
     // The table size after removing the index might not get back the original one,
-    // i.e. tableSizeWithDefaultIndex. Because entries in index_map file are reordered,
-    // and the file might get a different size. tableSizeAfterRemovingIndex should be
-    // close to tableSizeWithDefaultIndex, but their relationship is not deterministic.
+    // i.e. tableSizeWithDefaultIndex. Because indices in column.psf file are reordered,
```

Comment:
After index removal the size should be deterministic. We can store a member variable for the size, and compare that across these triggering tests

Reply:
size of columns.psf is deterministic, but the table size also includes the index_map file, whose size is not deterministic because the values of startOffset and size are changed when placement of indices in columns.psf change.

Reply:
When the index is removed, then the index order becomes deterministic, and we can assert on the size across the tests